leafsmart-front-end/src/components/EventsList.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default function EventListItem(props) {
```

Comment:
The filename and the component being different is wigging out my brain a smidge. Could we make them the same? 
(Maybe the <Events /> component holds a bunch of <EventCard />s?)